scripts/exsrw_aqm_ics.sh

```diff
@@ -8,7 +8,11 @@
 #-----------------------------------------------------------------------
 #
 . ${USHsrw}/source_util_funcs.sh
-source_config_for_task "task_aqm_ics" ${GLOBAL_VAR_DEFNS_FP}
+for sect in user nco platform workflow nco global verification cpl_aqm_parm \
+  constants fixed_files grid_params \
+  task_aqm_ics ; do
```

Comment:
There is no `task_aqm_ics` in `config_defaults.yaml`.

```suggestion
  constants fixed_files grid_params ; do
```